### DISCUSSION TITLE
set swiperSlide `height: auto` using the angular package

### DISCUSSION THREAD

**Discussion by onmax** [nolimits4web/swiper#4372] [Q&A] [answered]
Hey guys,

I just recently start using Swiper with Angular although I am not new to Swiper. Right now I have a project were I have the following  swiper component:
```html
<swiper class="font-sans w-full flex-1" [allowTouchMove]="false">
        <!-- First slide -->
        <ng-template class="flex flex-col overflow-auto" swiperSlide>
         
        <!-- More slides -->
</swiper>
```

Everything until this point works perfectly fine as expected. The problem is that I want to change the property of the `div.swiper-slide` to `height: auto` instead of `height: 100%` which is the default value. I tried using `style="height: auto !important"` or using a class with `.swiper-slide__h-auto { height: auto !important }` but nothing works because I cannot overwrite the CSS property.

Here I show a picture where the `height` property is not being overwritten:
HTML:
```html
<!-- First slide -->
        <ng-template class="h-full flex flex-col overflow-auto" swiperSlide>
```
![image](https://user-images.githubusercontent.com/22072217/112494904-b0b7cd80-8d83-11eb-824c-91ce1852b4b9.png)

Any ideas?

**Comment by nolimits4web** [comment]
Apparently it happens because of the order how your CSS included. Or just add custom class to swiper-slide with `height: auto !important`

**Comment by nolimits4web** [comment, marked as answer]
Or make sure your CSS files included AFTER swiper's styles

**Reply by onmax** [reply]
Yeah, I think I am stupid. I didn't thing about that. It should've been the first thing to check. Thanks for the quick response. And also, I want to thank you for the great job you have done with swiper, I have use it in some many websites!!!
